Deployment/src/Modules/AppInsights/main.tf

```
@@ -0,0 +1,6 @@
+resource "azurerm_application_insights" "app_insights" {
+  name                = var.name
+  location            = var.location
+  resource_group_name = var.resource_group_name
+  application_type    = "web"
+}
```

This resource is missing tags

```suggestion
  tags                = var.tags
}
```

---

Added missing tag